{
	"name": "CVE-2018-14034",
	"seq": "2018-14034",
	"type": "CAN",
	"status": "Candidate",
	"phase": {
		"text": "Assigned",
		"date": "20180712"
	},
	"desc": "An issue was discovered in the HDF HDF5 1.8.20 library. There is an out of bounds read in the function H5O_pline_reset in H5Opline.c.",
	"refs": [
		{
			"text": "https://github.com/TeamSeri0us/pocs/blob/master/hdf5/README2.md",
			"source": "MISC",
			"url": "https://github.com/TeamSeri0us/pocs/blob/master/hdf5/README2.md"
		}
	],
	"votes": {}
}
